neuralmonkey/learning_utils.py

```diff
@@ -209,6 +209,13 @@ def training_loop(tf_manager: TensorFlowManager,
                                         tf_manager.best_score_batch),
                                 color='blue')
 
+                            # store also graph parts
+                            all_coders = set.union(
+                                *[rnr.all_coders for rnr in runners])
+                            for coder in all_coders:
+                                for session in tf_manager.sessions:
+                                    coder.save(session)
```

Comment:
ted se to uklada vzdy, to jsem prehlednul. Kazdopadne nevim jake modely trenujete vy, ale myslim, ze kdyz se jednou za pul hodiny neco ulozi tak to nebude tak pomale, nebo to je fakt az tak pomale (predpokladal jsem tak do sekundy max nekolik sekund)